### ISSUE TITLE
Leaflet timeDimension WMS

### ISSUE THREAD

**Issue by ocefpaf** [python-visualization/folium#328]
We are using Leaflet timeDimension for `TimestampedGeoJson` plugin, but we could also add the WMS functionality:

``` js
var testWMS = "http://thredds.socib.es/thredds/wms/observational/hf_radar/hf_radar_ibiza-scb_codarssproc001_aggregation/dep0001_hf-radar-ibiza_scb-codarssproc001_L1_agg.nc"
var testLayer = L.nonTiledLayer.wms(testWMS, {
    layers: 'sea_water_velocity',
    format: 'image/png',
    transparent: true,
    styles: 'prettyvec/rainbow',
    markerscale: 15,
    markerspacing: 20,
    abovemaxcolor: "extend",
    belowmincolor: "extend",
    colorscalerange: "0,0.4",
    attribution: 'SOCIB HF RADAR | sea_water_velocity'
});
var testTimeLayer = L.timeDimension.layer.wms(testLayer);
testTimeLayer.addTo(map);
```

I am just not sure if we should overload the standard WMS functionality with these options or create a separate plugin.

**Comment by acrosby** [comment]
I am interested in this functionality being added. I would recommend adding WmsTimeTileLayer. It would also be great to be able to add additional parameter arguments (like for ncwms) as a dict to the wms layer objects instead of putting them in the wms url like I am doing now as a work around.

**Comment by ocefpaf** [comment]
Indeed those are nice ideas @acrosby. Are you up for a PR :wink:

**Comment by acrosby** [comment]
I'll take a look at it.

**Comment by acrosby** [comment]
I made both of these changes in PR https://github.com/python-visualization/folium/pull/644

Implemented as a "plugin" like TimestampedGeoJSON, but with a class that takes a WmsTileLayer as an argument.

**Comment by ocefpaf** [comment]
Awesome! Thanks @acrosby!!

I will try to review this over the weekend. If not please bug me! I don't want this feature to slip under the cracks.

**Comment by ocefpaf** [comment]
Implemented in #644
